random.js

```diff
@@ -0,0 +1,76 @@
+var slideIndex = 0;
+var slides = document.getElementsByClassName("mySlides");
+var dots = document.getElementsByClassName("slide-inactive");
+var i;
+var userClick = false;
+let myTimeout;
+
+function currentSlide(n){
+    showSlides(slideIndex = n);
+}
+
+let showSlides = n => {
+    slideIndex > slides.length ? slideIndex = 1 : false;
+    n < 1 ? slideIndex = slides.length : false;
+
+    for(i = 0; i < slides.length; i++){
+        slides[i].style.display = "none";
```

Comment:
You can make this a lot easier.

```
slides.forEach((slide) => {
  slide.style.display = "none"
}
```

Reply:
I keep getting "slides.forEach is not a function
    at showSlides (index.js:27)"

Reply:
Is slides an array? If not, you shouldn't be iterating over it like that anyway.

Out of curiosity, try
```
[...slides].forEach((slide) => {
  slide.style.display = "none"
}
```

Reply:
oooo Interesting. That way works.